### ISSUE TITLE
[3.0] editor not remembering layout

### ISSUE THREAD

**Issue by Gibbz** [godotengine/godot#8701]
**Operating system or device - Godot version:**
ubuntu + arch

**Issue description:**

the editor doesnt remember the layout

**Steps to reproduce:**
resize window, change filesystem type from grid to list view. 
Close and reopen godot editor.
Changes are back to the original state.

**Link to minimal example project:**

**Comment by Gibbz** [comment]
and what do you know. Just now it saved the settings! Weird! feel free to close this issue if you cant reproduce :)

**Comment by bojidar-bg** [comment]
@Gibbz Are you sure you opened the same project you closed the previous time? Since, from my experience, Godot saves layouts per-project and not globally.

**Comment by Gibbz** [comment]
Yep same project, just seems to be intermittent. But it did save the layout
just after posting so I dont know.

On 10 May 2017 12:01 am, "Bojidar Marinov" <notifications@github.com> wrote:

@Gibbz <https://github.com/Gibbz> Are you sure you opened the same project
you closed the previous time? Since, from my experience, Godot saves
layouts per-project and not globally.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/godotengine/godot/issues/8701#issuecomment-300183458>,
or mute the thread

**Comment by kubecz3k** [comment]
First of all thank you for your report and sorry for the delay.

We released Godot 3.0 in January 2018 after 18 months of work, fixing many old issues either directly, or by obsoleting/replacing the features they were referring to.

We still have hundreds of issues whose relevance/reproducibility needs to be checked against the [current stable version](https://godotengine.org/download), and that's where you can help us.
**Could you check if the issue that you described initially is still relevant/reproducible in Godot 3.0 or any newer version, and comment about its current status here?**

For bug reports, please also make sure that the issue contains detailed steps to reproduce the bug and, if possible, a zipped project that can be used to reproduce it right away. This greatly speeds up debugging and bugfixing tasks for our contributors.

Our Bugsquad will review this issue more in-depth in 15 days, and potentially close it if its relevance could not be confirmed.

Thanks in advance.

*Note: This message is being copy-pasted to many "stale" issues (90+ days without activity). It might happen that it is not meaningful for this specific issue or appears oblivious of the issue's context, if so please comment to notify the Bugsquad about it.*
